#! /bin/sh /usr/share/dpatch/dpatch-run
## htmlgen-fakeroot.dpatch by  <debian@adkgis.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: use fakeroot to avoid weird stat/permission problem when
## DP: building under pbuilder. This was preventing most man
## DP: pages from being built.

@DPATCH@
diff -urNad grass-6.3~20070922~/include/Make/Html.make grass-6.3~20070922/include/Make/Html.make
--- grass-6.3~20070922~/include/Make/Html.make	2007-05-12 06:20:06.000000000 +0200
+++ grass-6.3~20070922/include/Make/Html.make	2007-09-26 11:25:39.000000000 +0200
@@ -44,7 +44,7 @@
 	GISBASE=$(RUN_GISBASE) \
 	PATH="$(BIN):$$PATH" \
 	$(LD_LIBRARY_PATH_VAR)="$(BIN):$(ARCH_LIBDIR):$($(LD_LIBRARY_PATH_VAR))" \
-	LC_ALL=C $(1) --html-description | grep -v '</body>\|</html>' > $(PGM).tmp.html ; true
+	LC_ALL=C fakeroot $(1) --html-description | grep -v '</body>\|</html>' > $(PGM).tmp.html ; true
 
 # html rules for cmd commands
 htmlcmd: htmlcmd1 htmlgen
